src/app/sharedUI/screens/Artwork/Artwork.tsx

```diff
@@ -48,7 +48,7 @@ export const Artwork = () => {
 
   const height = Dimensions.get("window").height
 
-  const snapPoints = useMemo(() => [height > 800 ? "65%" : "75%"], [])
+  const snapPoints = useMemo(() => [height > 1000 ? "45%" : height > 800 ? "65%" : "75%"], [])
```

Comment:
```suggestion
  const snapPoints = useMemo(() => [500], [])
```
This will make it look nicer on all devices. We dont need height.

Reply:
if 500 is enough for both device types sure but either way seems okay to me if it is better to be dynamic based on. device size